minimesos/src/main/java/com/containersol/minimesos/docker/DockerContainersUtil.java

```diff
@@ -256,4 +229,26 @@ public static Container getContainer(String containerId) {
         return container;
     }
 
+    private static class PullImageResultCallback extends com.github.dockerjava.core.command.PullImageResultCallback {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS2176)